### PR TITLE
feat: Add staging disclaimer

### LINKED ISSUE #813: Staging disclaimer
### Describe the feature

Create a modal that acts as a disclaimer on our public staging instance. The disclaimer should be toggled on-off via the config. Default on. The disclaimer should only be visible every 24h after the disclaimer got accepted.

- Modal title:
  - 🇺🇸 `Development and preview instance`
  - 🇩🇪 `Entwicklungs- und Vorschauinstanz`
- Modal description:
  - 🇺🇸 `This public instance of helpwave tasks is for development and preview purposes. Please make sure to enter non-confidential testing data. This instance can be wipped out at any time.`
  - 🇩🇪 `Diese öffentliche Instanz von helpwave tasks ist für Entwicklungs- und Vorschauzwecke gedacht. Bitte stellen Sie sicher, dass Sie nicht-vertrauliche Testdaten eingeben. Diese Instanz kann jederzeit ausgelöscht werden.`
- One green accept button that closes the modal. Only this one button closes the modal.
- Modal footer contains links to our imprint and privacy notices of our website

### Why is this feature needed?

We need to make sure that everyone that is using staging is aware of the environment via an required opt-in.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by JannikWibker**
## Which issues does this pull request close?
closes #813 

Currently still missing:
- [ ] One green accept button that closes the modal. Only this one button closes the modal.
- [ ] Modal footer contains links to our imprint and privacy notices of our website

### CODE REVIEW

**Comment by MaxSchaefer** [comment]
@JannikWibker any progress here?

**Comment by DasProffi** [comment]
@MaxSchaefer @use-to Which imprint and privacy notice do we use here?

**Comment by MaxSchaefer** [comment]
> @MaxSchaefer @use-to Which imprint and privacy notice do we use here?

https://cdn.helpwave.de/imprint.html
https://cdn.helpwave.de/privacy.html (In progress but can already be linked)

File: tasks/pages/index.tsx [hunk 1]
```diff
@@ -41,13 +47,35 @@ const Dashboard: NextPage<PropsWithLanguage<DashboardTranslation, DashboardServe
...
+  const [lastTimeStagingDisclaimerDismissed, setLastTimeStagingDisclaimerDismissed] = useLocalStorage('staging-disclaimer-dismissed-time', 0)
+
+  const dismissStagingDisclaimer = () => {
+    setLastTimeStagingDisclaimerDismissed(new Date().getTime())
+    setStagingDiclaimerOpen(false)
+  }
+
+  useEffect(() => {
+    const ONE_DAY = 1000 * 60 * 60 * 24
+    if (new Date().getTime() - lastTimeStagingDisclaimerDismissed > ONE_DAY && config.showStagingDisclaimerModal) {
```
Comment (MaxSchaefer) [inline tasks/pages/index.tsx:60]:
```suggestion
    if (config.showStagingDisclaimerModal && new Date().getTime() - lastTimeStagingDisclaimerDismissed > ONE_DAY) {
```

**Review by MaxSchaefer — APPROVED** [review]

**Comment by DasProffi** [comment]
@MaxSchaefer the deploy to cloudflare fails

**Pull request merged**